src/main/target/YUPIF4/target.h

```diff
@@ -88,8 +88,8 @@
 #define UART6_TX_PIN            PC6
 
 #define USE_SOFTSERIAL1
-#define SOFTSERIAL1_RX_PIN      PB0 // PWM5
-#define SOFTSERIAL1_TX_PIN      PB1 // PWM7
+#define SOFTSERIAL1_RX_PIN      PB1 // PWM7
```

Comment:
Is this swap actually needed? If not I'd revert it, to avoid tripping users with existing wiring.

Reply:
No users have currently access to PB1 and thus can use the softserial for half duplex protocols. With this change, users will have the possibility to use the motor 5 output for softserial Tx, to connect the smartport for example.

Reply:
I see.